include/sound/sof/topology.h

```diff
@@ -57,8 +57,7 @@ struct sof_ipc_comp {
 	uint32_t pipeline_id;
 	uint32_t core;
 
-	/* reserved for future use */
-	uint32_t reserved[1];
+	uint32_t uid;	/**< Address of uuid_entry */
```

Comment:
This should be squashed with the previous patch no?

Reply:
This is actually the first commit of the series (except the one for smart_amp), it adding the uid for the generic sof_ipc_comp (and can be used for non-process/effect components in the future also) so I think it should be a separated one(not quashed with the token one or the generic process component one).

Reply:
it is ordered as the last one here, maybe you committed them in a wrong order?

Reply:
No idea what this means @keyonjie: a uint32_t to represent an address?

Reply:
> it is ordered as the last one here, maybe you committed them in a wrong order?

It is github display issue, see it here:
https://github.com/keyonjie/linux/commits/uuid

Reply:
> No idea what this means @keyonjie: a uint32_t to represent an address?

Yes, it represents uuid entry inside the .static_uuids section, see it here:
https://github.com/orgs/thesofproject/teams/sof-developers/discussions/36/comments/10

Reply:
> > it is ordered as the last one here, maybe you committed them in a wrong order?
> 
> It is github display issue, see it here:
> https://github.com/keyonjie/linux/commits/uuid

@lyakh I just 'reword' those commits so now you should see the correct sequence, the bad thing is that you have to 'reword' it every time before pushing. :(

Reply:
Try to do something like this for each commit in sequence: `git commit --amend --reset-author --no-edit`. It reset commit date, by which commits are sorted in GH

Reply:
@ktrzcinx Thanks that's another work-around.

Reply:
> It is github display issue, see it here:

When force-pushing you apparently can't trust the order of commits (among other issues)
https://github.com/zephyrproject-rtos/zephyr/pull/14444#issuecomment-613026905

Reply:
The implementation is changed and this is not existed anymore, mark it "resolved".